pydantic/mypy.py

```diff
@@ -68,7 +68,8 @@
 BASESETTINGS_FULLNAME = 'pydantic.env_settings.BaseSettings'
 FIELD_FULLNAME = 'pydantic.fields.Field'
 DATACLASS_FULLNAME = 'pydantic.dataclasses.dataclass'
-BUILTINS_NAME = 'builtins' if float(mypy_version) >= 0.930 else '__builtins__'
+MYPY_VERSION = tuple(int(part) for part in mypy_version.split('+')[0].split('.'))
+BUILTINS_NAME = 'builtins' if MYPY_VERSION >= (0, 930) else '__builtins__'
```

Comment:
Please move this into a function so we can test some real life values.

Reply:
Like that?